pom.xml

```diff
@@ -60,6 +60,11 @@
                 <artifactId>org.wso2.carbon.logging</artifactId>
                 <version>${commons-logging.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.ops4j.pax.logging</groupId>
+                <artifactId>pax-logging-api</artifactId>
+                <version>${pax.logging.api.version}</version>
+            </dependency>
```

Comment:
let's use the wso2 pax logging dependency which does not embed log4j1
https://github.com/wso2/wso2-pax-logging/tree/1.11.x